Intro to css values (#44937)
mdn/content: files/en-us/web/css/reference/properties/font-size/index.md

### Values

> This property is specified as a `<length>`, a `<percentage>`, or one of the following keyword values:

Reviewer: ```suggestion
This property is specified as a single value from the following list:
```
